ui.apps/src/main/content/jcr_root/apps/asset-share-commons/components/search/results/result/card/templates/card.html

```diff
@@ -27,9 +27,22 @@
             class="ui card cmp-card">
 
         <a class="image cmp-image__wrapper--card" href="${assetDetails.url @ suffix = asset.path}">
-            <img src="${asset.properties['thumbnail'] || properties['missingImage'] @ context = 'attribute'}"
-                 class="cmp-image--card"
-                 alt="${asset.properties['title']}"/>
+        
+        	<sly data-sly-test="${asset.properties['type'] == 'CONTENT-FRAGMENT'}">
+				<img src="${asset.properties['thumbnail']}"
+				     class="cmp-image--card"
+				     alt="${asset.properties['title']}"
```

Comment:
Lets break out the content region by high level asset type instead of having multiple checks for Content Fragment. Can do it via templates similar to [0] and [1] with another file called "asset-types.html". It doesn't have to be a 1:1 mapping. I think 2 templates should work for now, a default group and then a "group" for CF+Document+Presentation since Document/Presentation also do not have Resolution.

![image](https://user-images.githubusercontent.com/8974514/40268261-3a1f37f6-5b38-11e8-97a9-51ffb4680599.png)

[0] https://github.com/Adobe-Marketing-Cloud/asset-share-commons/blob/develop/ui.apps/src/main/content/jcr_root/apps/asset-share-commons/components/details/metadata/metadata.html.

[1] https://github.com/Adobe-Marketing-Cloud/asset-share-commons/blob/develop/ui.apps/src/main/content/jcr_root/apps/asset-share-commons/components/details/metadata/templates/field-types.html

Reply:
@godanny86 , @davidjgonzalez 
Wondering how do I classify assets into default group(media types) and Documents. Using the same AssetType Computed Property ? If so, I think we should simply do away with the configurable labels